tests/micro/zephyr/test_zephyr_aot.py

```diff
@@ -214,6 +214,9 @@ def test_tflite(platform, west_cmd, skip_build, tvm_debug):
 
 
 def test_qemu_make_fail(platform, west_cmd, skip_build, tvm_debug):
+    if platform not in ["host", "mps2_an521"]:
```

Comment:
should we add a decorator to tvm.testing?

Reply:
There are more changes that need to be done here. I removed this changes and will send another PR.